guessit/test/streaming_services.yaml

```diff
@@ -0,0 +1,1148 @@
+  source: Web
+  streaming_service: CNBC
+  title: Presidential Debates
+  type: episode
+  video_codec: H.264
+
+? What.The.Fuck.France.S01E01.Le.doublage.CNLP.WEBRip.AAC2.0.x264-TURTLE
+: audio_channels: '2.0'
+  audio_codec: AAC
+  country: FR
+  episode: 1
+  episode_title: Le doublage
+  other: Rip
+  release_group: TURTLE
+  season: 1
+  source: Web
+  streaming_service: Canal+
+  title: What The Fuck
+  type: episode
+  video_codec: H.264
+
+? SuperMansion.S02.720p.CRKL.WEBRip.AAC2.0.x264-VLAD
+: audio_channels: '2.0'
+  audio_codec: AAC
+  other: Rip
+  release_group: VLAD
+  screen_size: 720p
+  season: 2
+  source: Web
+  streaming_service: Crackle
+  title: SuperMansion
+  type: episode
+  video_codec: H.264
+
+? Chosen.S02.1080p.CRKL.WEBRip.AAC2.0.x264-AJP69
+: audio_channels: '2.0'
+  audio_codec: AAC
+  other: Rip
+  release_group: AJP69
+  screen_size: 1080p
+  season: 2
+  source: Web
+  streaming_service: Crackle
+  title: Chosen
+  type: episode
+  video_codec: H.264
+
+? Chosen.S03.1080p.CRKL.WEBRip.AAC2.0.x264-AJP69
+: audio_channels: '2.0'
+  audio_codec: AAC
+  other: Rip
+  release_group: AJP69
+  screen_size: 1080p
+  season: 3
+  source: Web
+  streaming_service: Crackle
+  title: Chosen
+  type: episode
+  video_codec: H.264
+
+? Snatch.S01.1080p.CRKL.WEBRip.AAC2.0.x264-DEFLATE
+: audio_channels: '2.0'
+  audio_codec: AAC
+  other: Rip
+  release_group: DEFLATE
+  screen_size: 1080p
+  season: 1
+  source: Web
+  streaming_service: Crackle
+  title: Snatch
+  type: episode
+  video_codec: H.264
+
+? White.House.Correspondents.Dinner.2015.Complete.CSPN.WEBRip.AAC2.0.H.264-BTW
+: audio_channels: '2.0'
+  audio_codec: AAC
+  other:
+  - Complete
+  - Rip
+  release_group: BTW
+  source: Web
+  streaming_service: CSpan
+  title: White House Correspondents Dinner
+  type: movie
+  video_codec: H.264
+  year: 2015
+
+? The.Amazing.Race.Canada.S03.720p.CTV.WEBRip.AAC2.0.H.264-BTW
+: audio_channels: '2.0'
+  audio_codec: AAC
+  country: CA
+  other: Rip
+  release_group: BTW
+  screen_size: 720p
+  season: 3
+  source: Web
+  streaming_service: CTV
+  title: The Amazing Race
+  type: episode
+  video_codec: H.264
+
+? Miniverse.S01E01.Explore.the.Solar.System.2160p.CUR.WEB-DL.DDP2.0.x264-monkee
+: audio_channels: '2.0'
+  audio_codec: Dolby Digital Plus
+  episode: 1
+  episode_title: Explore the Solar System
+  release_group: monkee
+  screen_size: 2160p
+  season: 1
+  source: Web
+  streaming_service: CuriosityStream
+  title: Miniverse
+  type: episode
+  video_codec: H.264
+
+? Vixen.S02.720p.CWS.WEBRip.AAC2.0.x264-BMF
+: audio_channels: '2.0'
+  audio_codec: AAC
+  other: Rip
+  release_group: BMF
+  screen_size: 720p
+  season: 2
+  source: Web
+  streaming_service: CWSeed
+  title: Vixen
+  type: episode
+  video_codec: H.264
+
+? Abidin.Dino.DDY.WEBRip.AAC2.0.H.264-BTN
+: audio_channels: '2.0'
+  audio_codec: AAC
+  other: Rip
+  release_group: BTN
+  source: Web
+  streaming_service: Digiturk Diledigin Yerde
```

Comment:
@ratoaq2 @duramato i added as `'Digiturk Dilediğin Yerde'` but the special `g` is not showing up